examples/plot_visualize_bad_epochs.py

```diff
@@ -18,20 +18,21 @@
 # We will do this using ``openneuro-py`` which can be installed using pip
 # (``pip install openneuro-py``).
 
-import os
+import os.path as op
 import openneuro
-import autoreject
+import mne
 
 dataset = 'ds000117'  # The id code on OpenNeuro for this example dataset
 subject_id = 16  # OpenfMRI format of subject numbering
+run = 3  # the run number to use for this example
 
-target_dir = os.path.join(
-    os.path.dirname(autoreject.__file__), '..', 'examples', dataset)
-if not os.path.isdir(target_dir):
-    os.makedirs(target_dir)
-
-openneuro.download(dataset=dataset, target_dir=target_dir,
-                   include=[f'sub-{subject_id}/ses-meg/'])
+# make a temporary directory for the example, you'd put this somewhere
+# permanent on your computer for a real analysis
+target_dir = mne.utils._TempDir()
```

Comment:
I think there are two issues here:

1) we expose private class in MNE
2) this will redownload the dataset every time it's run

I think I prefer to have it so that it's downloaded in a folder in the examples dataset. Or if openneuro has some sane defaults, like in the home directory under a folder called "openneuro"